snapm/_snapm.py

```
@@ -1677,6 +1717,250 @@ def deactivate(self):
+    :rtype:    Optional[str]
+
+    :raises:   ValueError: If 'identifier' is empty or 'by_type' is not 'uuid' or 'label'.
+    :raises:   SnapmNotFoundError: If the 'blkid' command is not found on the system.
+    :raises:   SnapmSystemError: If the 'blkid' command exits with a non-zero status
+                                due to reasons other than the identifier not being found
+                                (e.g., permission issues).
+    :raises:   SnapmCalloutError: For any other unexpected errors during command execution
+                                  or parsing.
+    """
+    if by_type not in ["uuid", "label"]:
+        raise ValueError("Invalid 'by_type'. Must be 'uuid' or 'label'.")
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)